torch/distributed/elastic/timer/api.py

```diff
@@ -30,11 +33,20 @@ class TimerRequest:
     __slots__ = ["worker_id", "scope_id", "expiration_time"]
 
     def __init__(self, worker_id: Any, scope_id: str, expiration_time: float):
+        """
```

Comment:
Please remove the new dosctrings and only fix the errors listed in the issue. I think the issues clearly asks to skip the "Missing docstring" errors.